internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -381,6 +386,32 @@ func (r *ClusterDeploymentsReconciler) updateIfNeeded(ctx context.Context, clust
 	}
 	return update, reply, err
 }
+func (r *ClusterDeploymentsReconciler) updateInstallConfigOverrides(ctx context.Context, clusterDeployment *hivev1.ClusterDeployment,
+	cluster *common.Cluster) (bool, ctrl.Result, error) {
+	// handle InstallConfigOverrides
+	update := false
+	annotations := clusterDeployment.ObjectMeta.GetAnnotations()
+	installConfigOverrides := annotations[InstallConfigOverrides]
```

Comment:
Maybe need to skip if it not exists?

Reply:
If there is no annotation we need to make sure that there isn't installConfigOverride on the cluster...

Reply:
Right.